.github/workflows/terragrunt_plan_production.yml

```
@@ -44,6 +44,7 @@ env:
   TF_VAR_client_vpn_access_group_id: ${{ secrets.PRODUCTION_CLIENT_VPN_ACCESS_GROUP_ID }}
   TF_VAR_client_vpn_saml_metadata: ${{ secrets.PRODUCTION_CLIENT_VPN_SAML_METADATA }}
   TF_VAR_client_vpn_self_service_saml_metadata: ${{ secrets.PRODUCTION_CLIENT_VPN_SELF_SERVICE_SAML_METADATA }}
+  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ART_PAT }}
```

Maybe this is your tribute to Art in general?
```suggestion
  TF_VAR_gha_arc_pat: ${{ secrets.GHA_ARC_PAT }}
```